src/sidebar/services/groups.js

```diff
@@ -196,8 +191,25 @@ function groups(
       });
   }
 
+  const sortGroups = memoize(groups => {
```

Comment:
This is a bit awkward but seemed like the simplest way to make sure we maintained the same order as the old groups menu. Note because the new groups menu combines groups from different endpoints, it doesn't maintain the same ordering as the single groups.list endpoint.